platformio.ini

```diff
@@ -1,3 +1,8 @@
+[platformio]
+extra_configs =
```

Comment:
That is a good catch, I didn't know this feature! Maybe it would make sense to split config for each device into separate INIs down the line. Especially if PIO supports nested configuration (e.g. PLASMA config inherits from Device8M config, and another local PLASMAOTA config from that one)

Reply:
yeah, it can work this way pretty nice. I'll rework

Reply:
Can be (or better be) in a later PR at some day in the future